src/aspire/utils/bot_align.py

```diff
@@ -0,0 +1,181 @@
+"""
+This code is an ASPIRE port of Ruiyi Yang's `Bayesian Optimal Transport Alignment`.
+
+`https://github.com/RuiyiYang/BOTalign`
+
+`https://arxiv.org/pdf/2305.12310.pdf`
+"""
+
+import numpy as np
+import pymanopt
+from numpy.linalg import norm
+from scipy.optimize import minimize
+
+from aspire.operators import wemd_embed
+from aspire.utils.rotation import Rotation
+
+
+def align_BO(
+    vol_ref,
+    vol_given,
+    loss_type="wemd",
+    downsampling_level=32,
```

Comment:
Maybe something like `downsampled_size` would be more appropriate. “Level” is often associated with the inverse (i.e., high level of downsampling means smaller image).

Reply:
Sounds better, changed.